pants.toml

```diff
@@ -209,6 +209,9 @@ lockfile = "3rdparty/python/pyupgrade.lockfile"
 # For keep-runtime-typing to work we also can't use py30-plus
 args = ["--py39-plus", "--keep-runtime-typing"]
 
+[protoc]
+version = "3.20.1"
```

Comment:
It would be worth adding a comment here describing why we currently need to specify the version for `protoc`, even if it is the default value in Pants (which `3.20.1` currently is). We don't want somebody "helpfully" refactoring this entire stanza away.

Pants does provide a massive JSON dump of information that we can extract this information out of (even without an explicit configuration in `pants.toml`. The format is a little complex and I'm still working through it, but that seems like the ultimate way to approach this. In the meantime, of course, `grep` is perfectly fine; I really just care about a comment here at this stage. 😄